examples/authentication.ipynb

```diff
@@ -0,0 +1,70 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit ('harmony-py': venv)",
+   "metadata": {
+    "interpreter": {
+     "hash": "31862cc71836a94b0e0781803a3648767fc4cb197cc35bade0ddf231ddce7d7c"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "# Using Harmony-Py Authentication\n",
+    "\n",
+    "In `harmony-py` you instantiate a `Client` in order to communicate with Harmony. To make it easy to store your validated credentials `harmony-py` provides an `authenticate()` function which returns a session object to be passed as an argument to `Client`.\n"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# note: to run, you may want to create a soft link to the library for now: `ln -s ../harmony_py/ harmony_py`\n",
+    "# note: the keyword argument `verify` defaults to True, meaning it will verify the credentials are valid during session instantiation.\n",
+    "\n",
+    "from harmony_py.auth import authenticate\n",
```

Comment:
Do we want `from harmony_py` or `from harmony`?

Note: potential name clash with https://pypi.org/project/harmony-pytorch/

Reply:
I think I'd prefer we name the package `harmony` too.

Reply:
This is now done